docs/EXAMPLES.md

```diff
@@ -39,13 +39,13 @@ And we provide `ctr` subcommand `rpull` as plugin to use feature to support pull
 
 ```bash
 # you will see that the rpull doesn't pull layer data.
-sudo bin/ctr rpull overlaybd-registry.cn-hangzhou.cr.aliyuncs.com/example/redis:6.2.1_obd
+sudo bin/ctr rpull registry.hub.docker.com/overlaybd/redis:6.2.1_obd
 ```
 
 And start a container based on this image.
 
 ```bash
-$ sudo ctr run --net-host --snapshotter=overlaybd --rm -t overlaybd-registry.cn-hangzhou.cr.aliyuncs.com/example/redis:6.2.1_obd demo
+$ sudo ctr run --net-host --snapshotter=overlaybd --rm -t registry.hub.docker.com/overlaybd/redis:6.2.1_obd demo
```

Comment:
you can try `docker.io` instead of `registry.hub.docker.com`. it is shorter

Reply:
synced with @liulanzheng offline. the overlaybd need to handle 302 http code. keep `registry.hub.docker.com` here

Reply:
The support for 302 in overlaybd will be added soon

Reply:
It is not because 302, it is because overlaybd lack of the the support for registry mirrors. We will implement it soon.